### PR TITLE
Charts: describe custom styles options

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md [hunk 1]
```diff
@@ -10,4 +10,118 @@ Specifies a range container's background color.
...
+        </dxo-range-container>
+    </dx-{widget-name}>
+
+    <!-- tab: app.component.ts -->
+    // ...
+
+    export class AppComponent {
+        // ...
+        
+        customPattern = {
```
Comment (Krijovnick) [inline api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md]: Could we name it like backgroundColor or customColor? I don't like customPattern name because this object is not custom pattern, it contains custom pattern.

File: api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md [hunk 2]
```diff
@@ -10,4 +10,118 @@ Specifies a range container's background color.
...
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function(){
+        $("#{widgetName}Container").dx{WidgetName}({
+            // ...
+            rangeContainer: {
+                // ...
+                backgroundColor: {
+                    fillId: customPatternId
```
Comment (Krijovnick) [inline api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md:27]: Should we demonstrate where we get customPatternId from?

File: api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md [hunk 3]
```diff
@@ -10,4 +10,118 @@ Specifies a range container's background color.
...
+    const customPattern = {
+        fillId: customPatternId
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import {WidgetName}, { RangeContainer } from 'devextreme-vue/{widget-name}';
```
Comment (Krijovnick) [inline api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md]: devextreme-vue -> devextreme-react

File: api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/ranges/color.md [hunk 4]
```diff
@@ -7,4 +7,138 @@ type: String | ChartsColor
...
+    const customPattern = {
+        fillId: customPatternId
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import {WidgetName}, { RangeContainer, Range } from 'devextreme-vue/{widget-name}';
```
Comment (Krijovnick) [inline api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/ranges/color.md]: devextreme-vue -> devextreme-react

File: api-reference/10 UI Components/BaseGauge/5 Indicator Types/CommonIndicator/color.md [hunk 5]
```diff
@@ -7,4 +7,111 @@ type: String | ChartsColor
...
+    const customPattern = {
+        fillId: customPatternId
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import {WidgetName}, { ValueIndicator } from 'devextreme-vue/{widget-name}';
```
Comment (Krijovnick) [inline api-reference/10 UI Components/BaseGauge/5 Indicator Types/CommonIndicator/color.md]: There is many places with devextreme-vue instead of devextreme-react. Check them all and fix, please

File: api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md (hunk 2 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/backgroundColor.md:27]: I think it's not necessary, it will complicate code snippets. Users can go to registerPattern method and check it. The purpose of the color/backgroundColor code snippets is to show how to use the id.

File: includes/dataviz-polar-chart-series-color.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+        base: "#000000",
+        fillId: customPatternId
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import PolarChart, { Series } from 'devextreme-vue/chart';
```
Comment (Krijovnick) [inline includes/dataviz-polar-chart-series-color.md]: devextreme-react

File: includes/dataviz-charts-series-color.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+        base: "#000000",
+        fillId: customPatternId
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Chart, { Series } from 'devextreme-vue/chart';
```
Comment (Krijovnick) [inline includes/dataviz-charts-series-color.md]: devextreme-react

**Review by Krijovnick — APPROVED** [review]

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 8]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Registers a new gradient.
 
 ##### return: String
-<!-- Description goes here -->
+The fill id.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]: Since you use these very short descriptions, I would go without any article. 

You can say "Specifies the fill ID for this gradient." But when you only say "Fill ID", then you can go without an article. The article in the sentence tell us that it's "that exact fill id that belongs to THIS gradient". But when you only say one or two words that describe the kind of value - "width", "color", "fill id", I don't think you need articles. 

```suggestion
Fill ID.
```

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 9]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
...
 
 ##### param(type): String
-<!-- Description goes here -->
+The gradient type: 'linear' or 'radial'.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]:
```suggestion
Gradient type: 'linear' or 'radial'.
```

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 10]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
...
 
 ##### param(options): Object
-<!-- Description goes here -->
+The gradient's settings.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]:
```suggestion
Gradient settings.
```

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 11]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
...
 
 ##### field(options.colors): Array<GradientColor>
-<!-- Description goes here -->
+The array of gradient colors.
 
 ##### field(options.rotationAngle): Number
-<!-- Description goes here -->
+The rotation angle for linear gradients.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]:
```suggestion
Linear gradient rotation angle. 
```

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 12]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
...
-<!-- Description goes here -->
+The gradient type: 'linear' or 'radial'.
 
 ##### param(options): Object
-<!-- Description goes here -->
+The gradient's settings.
 
 ##### field(options.colors): Array<GradientColor>
-<!-- Description goes here -->
+The array of gradient colors.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]:
```suggestion
An array of gradient colors.
```

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 13]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
...
 
 ##### field(options.rotationAngle): Number
-<!-- Description goes here -->
+The rotation angle for linear gradients.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Implement this method to register a gradient id. Then, use the id in the component color.[fillId](/Documentation/ApiReference/Common_Types/charts/ChartsColor/#fillId) field.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]: Implement seems like the wrong word choice. They need to call the method, not implement it on their own. 

"Use in ... field" - I'd rather use a more direct verb "assign to ... field". Also, since we don't know which component we're talking about (could be any component), I would use indefinite article. 

```suggestion
Call this method to register a gradient ID. Assign the ID to a component's color.[fillId](/Documentation/ApiReference/Common_Types/charts/ChartsColor/#fillId) field.
```

File: api-reference/50 Common/utils/viz/registerGradient(type_options).md [hunk 14]
```diff
@@ -5,22 +5,185 @@ export: registerGradient
...
+                />
+            </Chart>
+        );
+    }
+
+---
+
+[note]
+
+- To correctly export a component with transparent gradient, specify gradient colors in the RGBA format.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerGradient(type_options).md]: We don't use articles with mass nouns like "format". 

```suggestion
- To correctly export a component with transparent gradient, specify gradient colors in RGBA format.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 15]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Registers a new pattern.
 
 ##### return: String
-<!-- Description goes here -->
+The fill id.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
Fill ID.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 16]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
...
 
 ##### param(options): Object
-<!-- Description goes here -->
+The pattern settings.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
Pattern settings.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 17]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
...
 
 ##### field(options.height): Number | String
-<!-- Description goes here -->
+The pattern height.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
Pattern height.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 18]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
...
 
 ##### field(options.template): function()
-<!-- Description goes here -->
+The SVG-based pattern template.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
SVG-based pattern template.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 19]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
...
 
 ##### field(options.width): Number | String
-<!-- Description goes here -->
+The pattern width.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
Pattern width.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 20]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
...
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Implement this method to register a pattern id. Then, use the id in the component color.[fillId](/Documentation/ApiReference/Common_Types/charts/ChartsColor/#fillId) field.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
Call this method to register a pattern ID. Assign the ID to a component's color.[fillId](/Documentation/ApiReference/Common_Types/charts/ChartsColor/#fillId) field.
```

File: api-reference/50 Common/utils/viz/registerPattern(options).md [hunk 21]
```diff
@@ -5,22 +5,191 @@ export: registerPattern
...
+                <CommonSeriesSettings 
+                    color={seriesColor}
+                />
+            </Chart>
+        );
+    }
+
+---
+
+[note] If you specify width and height of a pattern in percents, the exported component may look different.
```
Comment (Abadzhev) [inline api-reference/50 Common/utils/viz/registerPattern(options).md]:
```suggestion
[note] If you specify width and height of a pattern in percent, the exported component may look different. 
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md [hunk 22]
```diff
@@ -8,4 +8,119 @@ default: undefined
 Specifies the color of the series in the hovered state.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md:8]: You don't want to use an article with words like "state", "mode", or "format". 

```suggestion
Specifies the color of the series in hovered state.
```

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md [hunk 23]
```diff
@@ -8,4 +8,124 @@ default: undefined
 Specifies a series color.
 
 ---
-Use this property to paint all pie slices in one color.
\ No newline at end of file
+You can use this property to paint all pie slices in one color.
+
+#include common-colorlist
+
+You can also specify a custom pattern or gradient instead of a plain color. Implement the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to get an id. Then, do the following:
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md]: I would move the method call to the list too. 

```suggestion
You can also specify a custom pattern or gradient instead of a plain color. Follow the steps below: 

1. Call the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to obtain a fill ID.
```

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md [hunk 24]
```diff
@@ -8,4 +8,124 @@ default: undefined
...
+
+1. Specify the `base` color for labels and connectors in the **color** configuration object.
+
+1. Set the `fillId` field to the obtained id.
```
Comment (Abadzhev) [inline api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md]: I would maybe move this item to second position, to keep related items together (they both mention fill ID).

```suggestion
1. Set the `fillId` field to the obtained ID.
```

File: includes/dataviz-chartscolor.md [hunk 25]
```diff
@@ -0,0 +1 @@
+You can also specify a custom pattern or gradient instead of a plain color. Implement the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to get an id. Then, set the `fillId` field to the obtained id.
```
Comment (Abadzhev) [inline includes/dataviz-chartscolor.md]:
```suggestion
You can also specify a custom pattern or gradient instead of a plain color. Call the [registerPattern()](/Documentation/ApiReference/Common/Utils/viz/#registerPatternoptions) or [registerGradient()](/Documentation/ApiReference/Common/Utils/viz/#registerGradienttype_options) method to obtain a fill ID. Assign that value to the `fillId` field.
```

File: concepts/05 UI Components/Chart/10 Series/60 Customize Appearance.md [hunk 26]
```diff
@@ -0,0 +1,181 @@
+To change series color, implement one of the following options:
```
Comment (Abadzhev) [inline concepts/05 UI Components/Chart/10 Series/60 Customize Appearance.md]:
```suggestion
To change series color, use one of the following options:
```

File: concepts/05 UI Components/Chart/10 Series/60 Customize Appearance.md [hunk 27]
```diff
@@ -0,0 +1,181 @@
...
+
+- Specify a color [palette](/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes) for the component.
+- Specify [color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#color) for a series.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Chart/10 Series/60 Customize Appearance.md]:
```suggestion
- Specify a [color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#color) for a series.
```

File: concepts/05 UI Components/Chart/10 Series/60 Customize Appearance.md [hunk 28]
```diff
@@ -0,0 +1,181 @@
...
+- Specify [color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#color) for all series.
```
Comment (Abadzhev) [inline concepts/05 UI Components/Chart/10 Series/60 Customize Appearance.md]:
```suggestion
- Specify a [color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#color) for all series.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md (hunk 22 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md:8]: Noted

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md (hunk 23 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md]: Noted

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md (hunk 24 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md]: Noted

File: api-reference/10 UI Components/BaseGauge/5 Indicator Types/CommonIndicator/color.md [hunk 29]
```diff
@@ -7,4 +7,111 @@ type: String | ChartsColor
 Specifies a color of the indicator.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/BaseGauge/5 Indicator Types/CommonIndicator/color.md]:
```suggestion
Specifies the color of the indicator.
```

File: api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/ranges/color.md [hunk 30]
```diff
@@ -7,4 +7,138 @@ type: String | ChartsColor
 Specifies a color of a range.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/BaseGauge/1 Configuration/rangeContainer/ranges/color.md]:
```suggestion
Specifies the color of a range.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md [hunk 31]
```diff
@@ -5,7 +5,7 @@ default: 'solid'
 ---
 ---
 ##### shortDescription
-Specifies the dash style of the series line when the series is in the hovered state. Applies only to line-like series.
+Specifies the dash style of the series line when the series is in hovered state. Applies only to line-like series.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md]: Why did you remove 'the' here?

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/highlight.md [hunk 32]
```diff
@@ -5,7 +5,7 @@ default: true
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to lighten the series when a user points to it.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/highlight.md]: `when a user points to it` With what? A finger?

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/width.md [hunk 33]
```diff
@@ -5,7 +5,7 @@ default: 3
 ---
 ---
 ##### shortDescription
-Specifies the pixel-measured width of the series line when the series is in the hovered state.
+Specifies the pixel-measured width of the series line when the series is in hovered state.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/width.md]:
```suggestion
Specifies the pixel-measured width of the series line when the series is in the hovered state.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/border/color.md [hunk 34]
```diff
@@ -5,7 +5,7 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the color of the point border when the point is in the hovered state.
+Specifies the color of the point border when the point is in hovered state.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/border/color.md:8]: Why did you remove the article?

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/size.md [hunk 35]
```diff
@@ -5,7 +5,7 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specfies the diameter of series points in the hovered state.
+Specfies the diameter of series points in hovered state.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/size.md]: It should be either 'a hovered state' (if there is more than one hovered state) or 'the hovered state'

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md [hunk 36]
```diff
@@ -8,4 +8,126 @@ default: undefined
 Specifies a series color.
 
 ---
-Use this property to paint all pie slices in one color.
\ No newline at end of file
+You can use this property to paint all pie slices in one color.
+
+#include common-colorlist
+
+You can also specify a custom pattern or gradient instead of a plain color. Follow the steps below:
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md]:
```suggestion
You can also specify a custom pattern or gradient instead of a plain color:  
```

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md [hunk 37]
```diff
@@ -8,4 +8,126 @@ default: undefined
 Specifies a series color.
 
 ---
-Use this property to paint all pie slices in one color.
\ No newline at end of file
+You can use this property to paint all pie slices in one color.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/color.md]:
```suggestion
You can use this property to paint all pie slices in the same color.
```

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/hoverStyle/color.md [hunk 38]
```diff
@@ -8,4 +8,119 @@ default: undefined
 Sets a color for the series when it is hovered over.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/hoverStyle/color.md]:
```suggestion
Sets the color for the series when it is hovered over.
```

File: api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/selectionStyle/color.md [hunk 39]
```diff
@@ -8,4 +8,119 @@ default: undefined
 Sets a color for a series when it is selected.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxPieChart/5 Series Types/CommonPieChartSeries/selectionStyle/color.md]:
```suggestion
Sets the color for a series when it is selected.
```

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md (hunk 31 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md]: Because Vladimir Abadzhev told me so.

> You don't want to use an article with words like "state", "mode", or "format".

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/highlight.md (hunk 32 above)
Comment (vladaskorohodova) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/highlight.md]: With a mouse pointer

Comment (dirk-pieterse) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/highlight.md]:
```suggestion
Specifies whether to lighten the series when a user hovers the mouse pointer over it.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md (hunk 31 above)
Comment (dirk-pieterse) [reply api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md]: There is no mention of 'state' in their latest guideline on articles

**Review by dirk-pieterse — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md [hunk 40]
```diff
@@ -5,7 +5,122 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the color of the series in the hovered state.
+Specifies the color of the series in hovered state.
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/color.md]:
```suggestion
Specifies the color of the series in the hovered state.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md (hunk 31 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md]:
```suggestion
Specifies the dash style of the series line when the series is in the hovered state. Applies only to line-like series.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/border/color.md (hunk 34 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/border/color.md]:
```suggestion
Specifies the color of the point border when the point is in the hovered state.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/border/width.md [hunk 41]
```diff
@@ -5,7 +5,7 @@ default: 4
 ---
 ---
 ##### shortDescription
-Specifies the width of the point border when the point is in the hovered state.
+Specifies the width of the point border when the point is in hovered state.
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/border/width.md]:
```suggestion
Specifies the width of the point border when the point is in the hovered state.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/color.md [hunk 42]
```diff
@@ -5,10 +5,136 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the color of series points in the hovered state.
+Specifies the color of series points in hovered state.
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/color.md]:
```suggestion
Specifies the color of series points in the hovered state.
```

File: api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/size.md (hunk 35 above)
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/point/hoverStyle/size.md]:
```suggestion
Specfies the diameter of series points in the hovered state.
```
